### ISSUE TITLE
Enable AXE tool in the CI/CD process

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak-ui#4301]
### Description

As part of the company-wide effort to have RH products and services accessible, we plan to enable the AXE tool for critical and severe a11y violations.

**Comment by jonkoops** [comment]
If would be nice if we could somehow 'hook' into events that Cypress triggers (such as clicks). So we can run an accessibility check every time such an interaction happens. Otherwise we'd have to trigger these checks manually all over the place.
